src/admin/admin.module.ts

```diff
@@ -0,0 +1,18 @@
+import { Module } from '@nestjs/common';
+import { AdminController } from '../admin/admin.controller';
+import { UserModule } from '../user/user.module';
+import { AdminService } from '../admin/admin.service';
+import { TypegooseModule } from 'nestjs-typegoose';
+import { User } from '../model/user.model';
+@Module({
+    imports: [
+        TypegooseModule.forFeature([
+            { typegooseClass: User, schemaOptions: { timestamps: true } },
+        ]),
```

Comment:
This module does not require `User` model. Updating user status function should be implemented in `UserService`.